governance.md

```diff
@@ -83,111 +83,131 @@ Community.
-the project’s overall well-being. We want to encourage a diverse array of
+the project’s overall well-being.  We want to encourage a diverse array of
 backgrounds, viewpoints and talents in our team, which is why we explicitly do
 not define code as the sole metric on which council membership will be
 evaluated.
 
-If a Council member becomes inactive in the project for a period of one year,
-they will be considered for removal from the Council. Before removal, inactive
-Member will be approached by the BDFL to see if they plan on returning to
-active participation. If not they will be removed immediately upon a Council
-vote. If they plan on returning to active participation soon, they will be
-given a grace period of one year. If they don’t return to active participation
-within that time period they will be removed by vote of the Council without
-further grace period. All former Council members can be considered for
-membership again at any time in the future, like any other Project Contributor.
-Retired Council members will be listed on the project website, acknowledging
-the period during which they were active in the Council.
-
-The Council reserves the right to eject current Members, other than the BDFL,
-if they are deemed to be actively harmful to the project’s well-being, and
-attempts at communication and conflict resolution have failed.
+When invited to join the Steering Council Contributors are commiting to serve
+for 2 years.  At the end of the two years they may elect to, with consent of the
+rest of the council, re-join the council.  If they chose to not re-join the
+council the process above is used to recruit new members.
+
+The Steering Council members, other than the LD, will serve in 2 equal classes
+whose terms are offset by 1 year.  This will help preserve the continuity on the
+Steering Council over time.
+
+A Steering Council member can step down at anytime.  If a Council Member becomes
+inactive for a period of 2 months, they will be approached by the LD to see if
+they plan on returning to active participation.  If not they will be asked to
+step down, if the Council Member indicates they intend to be active again but
+have not done so after 1 month the Council my vote to remove them.
+
+If a Council Member leaves the council early they may be replaced, using the
+same process as above.  Their replacement will join the same class and serve the
+remainder of the 2 years.
+
+Each class can fluctuate between 2 and 3 members so long as the total council
+size (including the LD) is between 5 and 7.  If a class gets too small, an
+additional member must be recruited.
+
+All former Council members can be considered for membership again at any time in
+the future, like any other Project Contributor. Retired Council members will be
+listed on the project website, acknowledging the period during which they were
+active in the Council.
+
+The Council reserves the right to eject current Members, other than the LD, if
```

Comment:
fork.

Reply:
like as much as I was joking, I think it would be good to have an actual structural way to address BDFL/steering committee/etc problems. Not 'cause I think anybody is a problem, but b/c these are the sorts of things you'd rather write explicitly when you don't need them and never use than not write & have to figure out when you need them. I don't love forking as a solution 'cause I think a end goal of governance is also to mitigate potential fallout especially where the community is concerned.

Reply:
The governance already suggests forking if the  community disagrees with the direction the project is going, however on a bit more consideration, I added that the NF board can evict the PL as they are on the hook that our conduct is inline with their non-profit status.

Reply:
I'm still a bit unclear as to why in this governance structure the Steering council can evict each other but not the project lead given that it falls under the whole "steering the direction of the project" thing.